go/ethadapter/geth_rpc_client.go

```diff
@@ -69,7 +69,6 @@ func (e *gethRPCClient) Info() Info {
 }
 
 func (e *gethRPCClient) BlocksBetween(startingBlock *types.Block, lastBlock *types.Block) []*types.Block {
-	// todo (@matt) - this should be a stream
```

Comment:
only used in testing, this comment was from when this function drove the host catch-up